File: python/tests/exams/exams1/test_Exam3.py
from unittest import TestCase

from exams.exams1.Exam3 import roman_to_int


class TestExam(TestCase):

    def test_roman_to_int(self):
        self.assertEqual(3, roman_to_int("III"))
        self.assertEqual(4, roman_to_int("IV"))
        self.assertEqual(9, roman_to_int("IX"))
        self.assertEqual(58, roman_to_int("LVIII"))
        self.assertEqual(1994, roman_to_int("MCMXCIV"))

File: cpp/tests/exams/exams1/test_Exam3.cpp
#include "gtest/gtest.h"

#include "exams1.h"

TEST(Exam3, test1)
{
    ASSERT_EQ(3, romanToInt("III"));
    ASSERT_EQ(4, romanToInt("IV"));
    ASSERT_EQ(9, romanToInt("IX"));
    ASSERT_EQ(58, romanToInt("LVIII"));
}
